macos/Sources/Features/Terminal/TerminalController.swift

```diff
@@ -461,6 +461,31 @@ class TerminalController: BaseTerminalController {
         let data = TerminalRestorableState(from: self)
         data.encode(with: state)
     }
+    
+    func setInitialWindowPosition(_ window: NSWindow, x: Int16?, y: Int16?, windowDecorations: Bool) {
+        if let primaryScreen = NSScreen.screens.first {
+            let frame = primaryScreen.visibleFrame
+
+            if let windowPositionX = x, let windowPositionY = y {
+                // Offset titlebar if needed, otherwise use default padding of 12
+                // NOTE: Not 100% certain where this extra padding comes from but I'd love
```

Comment:
Highlighting this note here in case a reviewer has more knowledge in the mac window space, I couldn't find anything helpful here but my hunch is this is default gap/padding/margin for NSWindow frames